backend/types/post-Type.ts

```diff
@@ -0,0 +1,13 @@
+import { IUser } from './user-Type';
+
+export interface IPost extends Document {
+  _id?: string;
```

Comment:
make it type and why does it extend Document? who is document and rename we don't need I prefix